agent/src/agent/pipeline/manager.py

```diff
@@ -77,7 +74,7 @@ def show_preview(self):
         print(*errors, sep='\n')
 
 
-def get_sdc_creator(pipeline_obj: Pipeline) -> config_handlers.base.BaseConfigHandler:
+def get_config_handler(pipeline_obj: Pipeline) -> config_handlers.base.BaseConfigHandler:
```

Comment:
config handler name is too abstract, I wanted later to rename everything to sdc_something to indicate that this  code handles configuration in the streamsets